tests/cloner_test.go

```diff
@@ -0,0 +1,133 @@
+package tests
+
+import (
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"k8s.io/api/core/v1"
+
+	"kubevirt.io/containerized-data-importer/pkg/common"
+	"kubevirt.io/containerized-data-importer/pkg/controller"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+	"kubevirt.io/containerized-data-importer/tests/utils"
+)
+
+const (
+	testSuiteName        = "Cloner Test Suite"
+	namespacePrefix      = "cloner"
+	sourcePodFillerName  = "fill-source"
+	sourcePVCName        = "source-pvc"
+	fillData             = "123456789012345678901234567890123456789012345678901234567890"
+	testFile             = utils.DefaultPvcMountPath + "/source.txt"
+	fillCommand          = "echo \"" + fillData + "\" >> " + testFile
+	cloneCompleteTimeout = 10 * time.Second
+)
+
+var _ = Describe(testSuiteName, func() {
+	f, err := framework.NewFramework(namespacePrefix, &framework.Config{})
+	if err != nil {
```

Comment:
This `if` pattern will be repeated everywhere, which may be fine. What do you think of a new func named
`NewFrameworkOrDie` which returns only a *Framework and call panic if an error occurs?

Reply:
Indeed you are, and in the same spirit as I don't think we will have THAT many files, repeating the if err != nil code in here is not that bad.